class-gwiz-gf-feed-forge.php

```diff
@@ -279,6 +280,10 @@ public function modal_markup() {
 							}
 							?>
 						</div>
+						<div id="gfff-preview" style="border:1px solid #ccc;height:20px;width:100%;padding:2px;border-radius:4px;">
```

Comment:
For clarity, I'd rename this to `gfff-progress-bar` or something along those lines.

Reply:
Now I'm curious, do you think you could replace `#feeds_please_wait_container` with this progress bar?